CHANGELOG.md

```diff
@@ -45,6 +45,13 @@ Early Access Packages
 | Daml.Finance.Interface.Instrument.Option   |                    | 0.1.0          |
 | Daml.Finance.Interface.Instrument.Swap     | 0.2.0              | 0.2.1          |
 
+Experimental Packages
```

Comment:
I think this is the same as "Early Access Packages"

Reply:
I put the line under "Early Access Packages"